.swiftlint.yml

```diff
@@ -0,0 +1,6 @@
+opt_in_rules:
+    - force_unwrapping
```

Comment:
I’m OK with this rule as long as the linter also enforces that we assert or throw an error instead of failing silently. 😉

In practice, we’ll all have to be on the lookout for overuse of `guard let` and `if let` statements and `??` chaining. It’s very easy to get lazy with them, leading to mysterious and undebuggable problems down the line.

Reply:
This is the issue that I have with the `force_unwrapping` rule: There are plenty of times in which an IUO can be used appropriately, and it seems tedious to start doing workarounds (`guard let thing = potentialThing else { return assertionFailure("Bad wolf") }`) just to make a linter happy, and as @1ec5 pointed out, workarounds can be prone to introducing very nasty, hard-to-fix bugs.

The linter should be there to help guide us towards clean code. This seems to be a step in the wrong direction.

Reply:
I agree. There a many valid cases for `force_unwrapping`. I guess that's why it's disabled by default.